README.md

```diff
@@ -5,18 +5,17 @@ This repository allows you to install [Apache Airflow](https://airflow.apache.or
 ## Requirements
 - A supported operating system.
 - MySQL or PostgreSQL database in which to store Airflow metadata.
-- [Airflow](https://github.com/teamclairvoyant/apache-airflow-parcels) and [RabbitMQ](https://github.com/teamclairvoyant/rabbitmq-cloudera-parcel) parcels need to be installed.
 
 ### Currently Supported Versions of Airflow
-- Airflow 1.7.1.3
-- Airflow 1.8.0
+- Airflow 1.10
 
 ### Currently Supported Operating Systems
-- CentOS 6 & 7
-- RHEL 6 & 7
+- CentOS/RHEL 6 & 7
+- Debian 8
+- Ubuntu 14.04, 16.04, & 18.04
 
 ## Installing the CSD
-1. Download the Jar file.  [Airflow CSD](https://teamclairvoyant.s3.amazonaws.com/apache-airflow/cloudera/csd/AIRFLOW-1.8.0.jar)
+1. Download the Jar file.  [Airflow CSD](http://archive.clairvoyantsoft.com/airflow/csd/)
```

Comment:
Should we add a guide to identify which csd version corresponds to which airflow version?

Reply:
There is no direct correlation in the CSD/parcel spec to say that A requires B.  As we build out new parcels, we may have to have a guide.  Right now, CSD 2.2.0 and parcel 1.10.3_b2 is the only valid combination.  (All others under archive.clairvoyantsoft.com will be purged once I am happy with the testing.)

Reply:
OK if there's currently no requirements around which CSD is compatible with each Parcel then we should be good